lib/pliny/config_helpers.rb

```diff
@@ -50,12 +50,20 @@ def array(method = nil)
     end
 
     def rack_env
-      if pliny_env == "development" || pliny_env == "test"
+      if app_env == "development" || app_env == "test"
```

Comment:
Yeah... Now when you said that, I also think backward compatibility with `pliny_env` could be better. I'll improve the solution and will add more tests on cases when `PLINY_ENV` is still defined in the environment.

Thanks for the idea! 👍